spec/models/user_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe User do
+  subject { User.new(name: 'username', postscounter: 1) }
+
+  it 'User`s name should be present' do
+    subject.name = nil
+    expect(subject.valid?).to be false
+  end
+
+  it 'Posts counter should be greater or equal to 0' do
+    subject.postscounter = -1
+    expect(subject.valid?).to be false
+  end
```

Comment:
- [ ] I would recommend adding the test cases for the positive case as well so that we can check the application in all the cases to avoid any unexpected error while running the application.